pkg/controller/perconaservermongodb/volumes.go

```diff
@@ -212,7 +212,7 @@ func (r *ReconcilePerconaServerMongoDB) resizeVolumesIfNeeded(ctx context.Contex
 		return errors.Errorf("requested storage (%s) is less than actual storage (%s)", requested.String(), actual.String())
 	}
 
-	if requested.Cmp(configured) == 0 || requested.Cmp(actual) == 0 {
+	if requested.Cmp(configured) == 0 && requested.Cmp(actual) == 0 {
```

Comment:
why?

Reply:
For some reason, the `pvc-resize` test kept failing in this PR.

There was a case where statefulset had an updated resources (`configured` var), but pvcs didn't (`actual` var). In this case it was not possible to start a resize because `requested.Cmp(configured) == 0` was true while `requested.Cmp(actual) == 0` wasn't.

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/1587/commits/e95cda3b4f33ab468dbde1c33368f22bc0149bb3